mmpose/core/utils/regularizations.py

```diff
@@ -0,0 +1,86 @@
+from abc import ABCMeta, abstractmethod, abstractproperty
+
+import torch
+
+
+class PytorchModuleHookRegister(metaclass=ABCMeta):
```

Comment:
How about the class name `PytorchModuleHook`. I think it is more of a hook, than a register.